openwisp_controller/config/tasks_zerotier.py

```
@@ -122,12 +76,10 @@ def handle_api_call(self, fn, *args, send_notification=True, **kwargs):
                 raise e
```

Ruff (0.14.14)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)